octomap:
  octomap_topic: "/octomap_full"
  pointcloud_topic: "/$(arg name)/semantic_pcl"
  frame_id: "world"
  resolution: 0.3
  max_range: 10.0
  raycast_range: 0.5
  clamping_thres_min: 0.006692851
  clamping_thres_max: 0.993307149
  occupancy_thres: 0.5
  prob_hit: 0.731058579
  prob_miss: 0.268941421
  psi: 1
  phi: -0.3
  publish_2d_map: true
  min_ground_z: -1.5
  max_ground_z: 1.5
  consensus_weight: 0.8
  sub_timer: 10
  pub_timer: 10

# Camera intrinsic matrix parameters
camera:
  fx: 65.24951330239097
  fy: 65.24951330239097
  cx: 80.0
  cy: 60.0

  width: 160
  height: 120

semantic_pcl:
  color_image_topic: "/$(arg name)/camera/image"
  semantic_image_topic: "/$(arg name)/camera/semantic_image"
  depth_image_topic: "/$(arg name)/depth_camera/depth/image_rect_raw"
  point_type: 0
  frame_id: "$(arg name)/camera"
